sos/report/plugins/processor.py

```diff
@@ -48,15 +51,24 @@ def setup(self):
         self.add_cmd_output([
             "lscpu",
             "lscpu -ae",
-            "cpupower frequency-info",
-            "cpupower info",
-            "cpupower idle-info",
             "cpufreq-info",
             "cpuid",
             "cpuid -r",
-            "turbostat --debug sleep 10"
         ], cmd_as_tag=True)
 
+        if (isinstance(self.policy, UbuntuPolicy) and
+                self.policy.dist_version() == 24.10):
+            self.cpu_kmods = ['msr']
```

Comment:
I was testing this right now. GCE doesn't have fedora40, but tested fedora39 there and we don't see it there, I also tested it using centos-9-stream image on GCE too

Just a moment ago, I also tested in LXD VM for fedora40, and we don't see the same issue, where I was able to reproduce this locally on ubuntu 24.04

Furthermore, it seems like that the `msr` module may be built-in the RH flavors of kernel, so maybe that's the reason for it not needed to be loaded

```
[root@fedora40-01 ~]# modinfo msr
name:           msr
filename:       (builtin)
license:        GPL
file:           arch/x86/kernel/msr
description:    x86 generic MSR driver
author:         H. Peter Anvin <hpa@zytor.com>
```

Obviously, having better RH expertise here would be very valuable

Reply:
That tracks. I'm on F38 myself and wasn't seeing it get explicitly loaded, which makes sense, but was why I was asking. I was seeing a different *msr module being loaded which is what made me think of a packaging difference, but all set on my end.

Reply:
Differences in the kernel config based on my 2 VMs

fedora40
```
[root@fedora40-01 ~]# grep MSR /boot/config-6.8.7-300.fc40.x86_64
CONFIG_XEN_PV_MSR_SAFE=y
CONFIG_X86_DEBUGCTLMSR=y
CONFIG_X86_MSR=y
CONFIG_SCSI_ARCMSR=m
```

ubuntu24.04
```
root@noble01:~# grep MSR /boot/config-6.8.0-31-generic 
CONFIG_XEN_PV_MSR_SAFE=y
CONFIG_X86_DEBUGCTLMSR=y
CONFIG_X86_MSR=m
CONFIG_SCSI_ARCMSR=m
```

so, safe to say, that RH would follow that same process and potentially have the `msr` module built in. We may need a `>=` instead of `==` for the next release of ubuntu, but we could potentially wait till our nest dev release to pop out in the next few weeks to see if anything changes there and once we update the CI, and we could capture it there